AccountingSystem/ConfigTemplate.cfg

```diff
@@ -25,4 +25,14 @@ Services
       }
     }
   }
-}
\ No newline at end of file
+}
+Agents
+{
+  NetworkAgent
+  {
+    MaxCycles = 0
```

Comment:
Yes, but anyway from time to time this agent is going to be restarted, at a minimum at every new release.